homer0/projext-plugin-webpack
# feat(projext): implement the bundle analyzer plugin

### What does this PR do?

This is the implementation of homer0/projext#90. It uses the new `analyze` task (and flag) to enable the [webpack bundle analyzer plugin](https://yarnpkg.com/en/package/webpack-bundle-analyzer)

### How should it be tested manually?

In order to test this, you need to install `homer0/projext#next` and make sure you delete the copy of `projext` `npm`/`yarn` will install inside this package's `node_modules`.

Now:

```bash
projext analyze [target]
```

It should open the bundle analyzer :D.

And, of course...

```bash
yarn test
# or
npm test
```
